test/e2e/suite/conformance/certificatesigningrequests/tests.go

```diff
@@ -73,6 +73,7 @@ func (s *Suite) Define() {
 		})
 
 		type testCase struct {
+			name    string
```

Comment:
suggestion: is it worth adding a comment here which links back to this PR, explaining why this isn't a map? I know if I came across this randomly without having seen this PR, I'd wonder why!

Reply:
I added the comment